aws-java-sdk-core/src/test/java/com/amazonaws/http/HttpMethodNameTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 Diffblue Limited
+ *
+ * Diffblue Limited licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.amazonaws.http;
+
+import com.amazonaws.http.HttpMethodName;
+import com.diffblue.deeptestutils.Reflector;
```

Comment:
No reflection used in tests for this class so could remove.

Reply:
Removed.